applications/FSIApplication/tests/test_FSIApplication.py

```diff
@@ -41,6 +41,7 @@ def AssembleTestSuites():
     smallSuite.addTest(FSICouplingInterfaceTest('test_fsi_coupling_interface'))
     smallSuite.addTest(FSIProblemEmulatorTest('testFSIProblemEmulatorWithAitken'))
     smallSuite.addTest(FSIProblemEmulatorTest('testFSIProblemEmulatorWithMVQN'))
+    smallSuite.addTest(FSIProblemEmulatorTest('testFSIProblemEmulatorWithIBNMVQN'))
```

Comment:
maybe makes more sense to add them all at once, this way new test will not be forgotten to be added

Reply:
Done.